src/components/list/list-item.types.ts

```diff
@@ -40,8 +40,25 @@ export interface ListItem<T = any> {
      * List of actions to display as a menu at the end of the item
      */
     actions?: Array<MenuItem | ListSeparator>;
+
+    /**
+     *Component used to render in the list
+     */
+    component?: ListComponent;
 }
 
 export interface ListSeparator {
     separator: true;
 }
+
+export interface ListComponent {
+    /**
+     * Name of the component
+     */
+    name?: string;
+
+    /**
+     * Extra properties to give the component in addition to the properties
```

Comment:
In addition to what properties? This looks like a copy-paste error from Lime CRM Admin?